### ISSUE TITLE
NPE when clicking the icon on bottom right

### ISSUE THREAD

**Issue by AtticSoftware** [OpherV/gitflow4idea#245]
* **I'm submitting a ...**
  - [ X] bug report
  - [ ] feature request
  - [ ] puppy => You're not submitting a puppy. I already have one and he's [adorable](https://twitter.com/Opherv/status/974341249099542528)

* **What is the current behavior?**
NPE when clicking the icon on bottom right

* **Is this a bug? Sorry about that. If so give me explicit details how to reproduce:**

1. Open IntelliJ
2. Click on the icon on the bottom right
3. Profit

* **What is the expected behavior?**
No NPE

* **What is the motivation / use case for changing the behavior?**

* **Please tell me about your environment:**
  
  - Gitflow4idea version: 0.6.9
  - Gitflow version: 1.12.2
  
  - *IntelliJ Help -> about > click copy icon and paste here. Should look like this:*
  ~~~~
    IntelliJ IDEA 2019.2.1 (Ultimate Edition)
Build #IU-192.6262.58, built on August 20, 2019
Runtime version: 11.0.3+12-b304.39 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.6
GC: ParNew, ConcurrentMarkSweep
Memory: 1981M
Cores: 8
  ~~~~

update failed for ActionGroup: Track Feature (null)[Track Feature]

java.lang.NullPointerException
	at java.base/java.lang.String.contains(String.java:2036)
	at gitflow.GitflowBranchUtil.getRemoteBranchesWithPrefix(GitflowBranchUtil.java:166)
	at gitflow.actions.AbstractTrackAction.isActionAllowedForBranch(AbstractTrackAction.java:31)
	at gitflow.actions.AbstractBranchAction.update(AbstractBranchAction.java:39)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:176)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.doUpdate(ActionUpdater.java:378)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$null$1(ActionUpdater.java:78)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.callAction(ActionUpdater.java:106)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$new$2(ActionUpdater.java:79)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.update(ActionUpdater.java:364)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.expandGroupChild(ActionUpdater.java:231)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$doExpandActionGroup$19(ActionUpdater.java:215)
	at com.intellij.util.containers.ContainerUtil.concat(ContainerUtil.java:1609)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.doExpandActionGroup(ActionUpdater.java:215)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.expandActionGroup(ActionUpdater.java:139)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.expandActionGroup(ActionUpdater.java:131)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:50)
	at com.intellij.ui.popup.ActionStepBuilder.appendActionsFromGroup(ActionStepBuilder.java:112)
	at com.intellij.ui.popup.ActionStepBuilder.buildGroup(ActionStepBuilder.java:69)
	at com.intellij.ui.popup.PopupFactoryImpl$ActionGroupPopup.getActionItems(PopupFactoryImpl.java:268)
	at com.intellij.ui.popup.PopupFactoryImpl$ActionGroupPopup.createStep(PopupFactoryImpl.java:249)
	at com.intellij.ui.popup.PopupFactoryImpl$ActionGroupPopup.<init>(PopupFactoryImpl.java:189)
	at com.intellij.ui.popup.PopupFactoryImpl$ActionGroupPopup.<init>(PopupFactoryImpl.java:173)
	at gitflow.ui.GitflowWidget.getPopupStep(GitflowWidget.java:115)
	at com.intellij.openapi.wm.impl.status.IdeStatusBarImpl$MultipleTextValuesPresentationWrapper$1.onClick(IdeStatusBarImpl.java:652)
	at com.intellij.ui.ClickListener$1.mouseReleased(ClickListener.java:59)
	at java.desktop/java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:297)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6651)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3342)
	at java.desktop/java.awt.Component.processEvent(Component.java:6416)
	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5026)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4858)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2772)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4858)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:778)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:751)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:749)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:748)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:817)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:762)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:405)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:704)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:404)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)

**Comment by costalfy** [comment]
Same here with version 0.6.9 and IntelliJ IDEA 2019.2.1 (Ultimate Edition) Build #IU-192.6262.58

**Comment by DiesIrae** [comment]
+1 :-)
Webstorm 2019.2.1, 0.6.9 of this plugin.
Same trace.
It occurrenced when commiting I believe.

**Comment by OpherV** [comment]
I'm a bit confused what icon are you referring to?

**Comment by adamlalikos** [comment]
@OpherV seems to occur when just hitting the `Gitflow` menu:
![Screen Shot 2019-08-23 at 5 23 25 PM](https://user-images.githubusercontent.com/46462701/63624574-d3dc3300-c5ca-11e9-90a3-6fe3d6207879.png)

**Comment by OpherV** [comment]
That's odd. Is any of you using a public repo that I might use to reproduce?

**Comment by DiesIrae** [comment]
No public repo for me, sorry :-/

**Comment by adamlalikos** [comment]
@OpherV I just setup a public project you can use: https://github.com/adamlalikos/demo-project.git

**Comment by petyots** [comment]
Same here

**Comment by chopinx** [comment]
Same with me

**Comment by Jigsaw5279** [comment]
Same here. 
Error only occurs when not on a feature branch.

**Comment by r-da** [comment]
Same here.

**Comment by imifarago** [comment]
Same here with PHPStorm 2019.2 and 2019.2.1 also (on Linux)

**Comment by h1dden-da3m0n** [comment]
interestingly enough that this did not happen in the preveous version with just changing the META-INF 
plugin.xml version to get it working in 2019.2 🤔 (aka install 0.6.8 and apply this "fix" from [#241]( https://github.com/OpherV/gitflow4idea/issues/241#issuecomment-515444744))

**Comment by mofulvtu** [comment]
Same with me

*  Git Flow Integration: 0.6.9

*  Git version is 2.22.0

 * IntelliJ IDEA 2019.2.1 (Ultimate Edition)

**Comment by OpherV** [comment]
People no need for the "me too" comments, we've already identified that an issue exists and I'll take care of it

**Comment by Mojtaba-Shafaei** [comment]
I get the NullpointerException with PhpStorm 2019.2 & 2019.2.1 and gitFlow 0.6.9 when I click on gitFlow 

this is the stack trace:

> 
update failed for ActionGroup: Track Feature (null)[Track Feature]

java.lang.NullPointerException
	at java.base/java.lang.String.contains(String.java:2036)
	at gitflow.GitflowBranchUtil.getRemoteBranchesWithPrefix(GitflowBranchUtil.java:166)
	at gitflow.actions.AbstractTrackAction.isActionAllowedForBranch(AbstractTrackAction.java:31)
	at gitflow.actions.AbstractBranchAction.update(AbstractBranchAction.java:39)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:176)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.doUpdate(ActionUpdater.java:378)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$null$1(ActionUpdater.java:78)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.callAction(ActionUpdater.java:106)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$new$2(ActionUpdater.java:79)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.update(ActionUpdater.java:364)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.expandGroupChild(ActionUpdater.java:231)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$doExpandActionGroup$19(ActionUpdater.java:215)
	at com.intellij.util.containers.ContainerUtil.concat(ContainerUtil.java:1609)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.doExpandActionGroup(ActionUpdater.java:215)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.expandActionGroup(ActionUpdater.java:139)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.expandActionGroup(ActionUpdater.java:131)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:50)
	at com.intellij.ui.popup.ActionStepBuilder.appendActionsFromGroup(ActionStepBuilder.java:112)
	at com.intellij.ui.popup.ActionStepBuilder.buildGroup(ActionStepBuilder.java:69)
	at com.intellij.ui.popup.PopupFactoryImpl$ActionGroupPopup.getActionItems(PopupFactoryImpl.java:268)
	at com.intellij.ui.popup.PopupFactoryImpl$ActionGroupPopup.createStep(PopupFactoryImpl.java:249)
	at com.intellij.ui.popup.PopupFactoryImpl$ActionGroupPopup.<init>(PopupFactoryImpl.java:189)
	at com.intellij.ui.popup.PopupFactoryImpl$ActionGroupPopup.<init>(PopupFactoryImpl.java:173)
	at gitflow.ui.GitflowWidget.getPopupStep(GitflowWidget.java:115)
	at com.intellij.openapi.wm.impl.status.IdeStatusBarImpl$MultipleTextValuesPresentationWrapper$1.onClick(IdeStatusBarImpl.java:652)
	at com.intellij.ui.ClickListener$1.mouseReleased(ClickListener.java:59)
	at java.desktop/java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:297)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6651)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3342)
	at java.desktop/java.awt.Component.processEvent(Component.java:6416)
	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5026)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4858)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2772)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4858)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:778)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:751)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:749)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:748)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:817)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:762)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:405)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:704)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:404)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
>

**Comment by cawa-93** [comment]
+1

**Comment by OpherV** [comment]
Fix in 0.7.0 once it's out
